spec/spec_helper.rb

```diff
@@ -31,7 +31,11 @@ def serializable_body(*)
 
 module TestClient
   extend RSpec::SharedContext
-  let(:client) { Oktakit::Client.new(token: test_okta_token, organization: 'okta-test') }
+  let(:client) do
+    Oktakit::Client.new(token:        test_okta_token,
+                        organization: test_okta_organization,
+                        api_endpoint: test_okta_api_endpoint)
+  end
```

Comment:
I didn't spend too much time trying to figure out how to scrub the okta API URL, which would make creating VCRs much easier.